app/scripts/contentscript.js

```diff
@@ -94,4 +94,8 @@ $(function() {
     'autosave':      true,
     'enableHistory': true
   });
+
+  document.getElementById("run").title = "Press ALT+R to run quickly.";
```

Comment:
I don't think ` quickly` is really needed here, it seems a bit redundant? What do you think?